app/views/operations/new.html.erb

```diff
@@ -0,0 +1,30 @@
+<div class="groups-container">
+  <nav class="nav-container fixed">
+    <div class="nav transactions-nav">
+      <%= link_to groups_path do %>
+        <i class="fa-light fa-backward back" style="font-family: FontAwesome;"></i>
+      <% end  %>

```

Comment:
[OPTIONAL] Based on the user experience described in the project requirements, when the user clicks on the "Back" button (<) in the operations/new page, the user should navigate to the transactions page for that category.